src/app/reporting/Engine.cpp

```diff
@@ -124,6 +124,8 @@ CHIP_ERROR Engine::BuildSingleReportDataAttributeDataList(ReportDataMessage::Bui
             {
                 if (clusterInfo->IsAttributePathSupersetOf(*path))
                 {
+                    // SetDirty's path don't have the particular nodeId, need to reover nodeId from subscribed path
```

Comment:
typo 'reover'.

Maybe we can just say " Include nodeId into GlobalDirtySet path"

Reply:
Done, thanks